### ISSUE TITLE
Unicode support

### ISSUE THREAD

**Issue by p3k** [darrikonn/td-cli#24]
Installed td-cli in virtual env with `pip3 install td-cli`:

```sh
# pip3 list
Package    Version
---------- -------
pip        21.0.1
setuptools 56.0.0
td-cli     2.0.1
wheel      0.36.2
```

Not able to enter e.g. German umlaut (e.g. `ä` as in my name, Tobi Schäfer):

![image](https://user-images.githubusercontent.com/16631/114362753-f8a66500-9b77-11eb-8048-32918e1cec6c.png)

Not to mention emoji et al.

**Comment by darrikonn** [comment]
Hmm, what's the font you're using in your terminal?
And is this only happening in the interactive mode, or does it also happen when you list your todos?

**Comment by p3k** [comment]
the font is ubuntu mono regular.

it does not happen when i enter a todo with such “special” chars non-interactively:

![image](https://user-images.githubusercontent.com/16631/114668683-4bf7ef00-9d01-11eb-8f81-05010c4e203e.png)

(good to know there is a work-around.)

it’s also correctly displayed in interactive mode, then. but entering the same char interactively creates output as described before.

**Comment by darrikonn** [comment]
Ahh happens as well for me when adding a new todo interactively.
Will make a fix!
Thanks for the issue ;)

**Comment by darrikonn** [comment]
Fixed here:  https://github.com/darrikonn/td-cli/pull/26

And available in `td-cli 2.0.2`

**Comment by p3k** [comment]
it works now, thanks, i can enter the precious characters. however, td-cli crashes now when i press the delete (or an arrow) key while editing a todo… should i open a new issue?

**Comment by darrikonn** [comment]
I just reopened this one. I'll take a look tomorrow :).

**Comment by darrikonn** [comment]
Should be fixed in `td-cli 2.0.3`.
Thanks for testing it out and notifying the bugs!

**Comment by p3k** [comment]
awesome, works perfectly now. thanks again.
